src/schemas/PartnerTrack.ts

```diff
@@ -0,0 +1,4 @@
+interface PartnerTrackSchema{
+    name: String,
+    expertises: PartnerExpertiseSchema
```

Comment:
Aqui tem que ser uma lista / array pq uma Expertise contém multiplos Qualifiers

A mesma coisa se aplica a todas as outras instâncias aonde você referencia outro schema como valor de um atributo; Em todos os casos deve ser lista 

```suggestion
    expertises: PartnerExpertiseSchema[]
```